File: android/app/src/main/java/com/vegettable/app/model/MonthlyPrice.java
package com.vegettable.app.model;

import com.google.gson.annotations.SerializedName;

public class MonthlyPrice {
    @SerializedName("month")
    private String month;

    @SerializedName("avgPrice")
    private double avgPrice;

    @SerializedName("volume")
    private double volume;

    public String getMonth() { return month; }
    public double getAvgPrice() { return avgPrice; }
    public double getVolume() { return volume; }
}

File: android/app/src/main/java/com/vegettable/app/model/MonthlyPrice.kt
package com.vegettable.app.model

import com.google.gson.annotations.SerializedName

class MonthlyPrice {
    @JvmField
    @SerializedName("month")
    val month: String? = null

    @JvmField
    @SerializedName("avgPrice")
    val avgPrice: Double = 0.0

    @SerializedName("volume")
    val volume: Double = 0.0
}
